linux/src/async_socket_linux.c

```diff
@@ -117,10 +117,6 @@ static int thread_worker_func(void* parameter)
                 if (recv_context != NULL)
                 {
                     recv_context->on_receive_complete(recv_context->on_receive_complete_context, ASYNC_SOCKET_RECEIVE_ABANDONED, 0);
-
-                    (void)interlocked_decrement(&recv_context->async_socket->pending_api_calls);
```

Comment:
>pending_api_calls [](http://example.com/codeflow?start=77&length=17)

as there are no specs, what's there to review...


signing off

Reply:
Specs are coming soon.